web/src/services/gql/fragments/story.ts

```diff
@@ -0,0 +1,12 @@
+import { gql } from "@apollo/client";
+
+export const storyFragment = gql`
+  fragment StoryFragment on Story {
+    id
+    title
+    pages {
+      id
```

Comment:
You have id here and on the storyPageFragment. Is that on purpose, or we can remove here?

Reply:
Thank you, fixed in https://github.com/reearth/reearth/pull/592/commits/4437e0d873eeea14c5b1c7b68c992f05e75296e0